lib/crypto/sr25519/sr25519_test.go

```diff
@@ -119,3 +123,54 @@ func TestNewKeypairFromMnenomic(t *testing.T) {
 	_, err = NewKeypairFromMnenomic(mnemonic, "")
 	require.NoError(t, err)
 }
+
+func TestVerifySignature(t *testing.T) {
```

Comment:
Run the parent test in parallel with

```suggestion
func TestVerifySignature(t *testing.T) {
	t.Parallel()
	
```